### ISSUE TITLE
tkinter.Canvas.coords doesn't accept item id like it should

### ISSUE THREAD

**Issue by Akuli** [python/typeshed#6920]
https://github.com/python/typeshed/blob/5d206358907c419802cd9afcfcc83cce4f01bebb/stdlib/tkinter/__init__.pyi#L1070-L1075

Looking at the `canvas` manual page, the first argument seems to be `tagOrId` which is missing here.

**Comment by Akuli** [comment]
Here's a minimal-ish example that displays a circle at runtime and creates a mypy error when type checking:

```
from tkinter import Canvas
c = Canvas()
c.pack()
circle_id = c.create_oval(0,0,0,0)
c.coords(circle_id, (50, 50, 100, 100))
c.mainloop()
```

**Comment by AlexWaygood** [comment]
@Akuli, has this been fixed by #7246?

**Comment by Akuli** [comment]
Yes, this has been fixed.
